onnxruntime/core/framework/allocation_planner.cc

```diff
@@ -829,6 +829,37 @@ class PlannerImpl {
     return p_provider->GetOrtDeviceByMemType(utils::IsInputOnCpu(node, &kernel_create_info, input_index) ? OrtMemTypeCPUInput : OrtMemTypeDefault);
   }
 
+  std::vector<std::pair<int, int>> GetAliasMap(const Node& node, const KernelCreateInfo& kernel_create_info) {
+    if (kernel_create_info.kernel_def != nullptr) {
```

Comment:
Dont need this if, this condition is already being checked on line 319

Reply:
Ok I see this is being called from 2 places. I call site checks this already and 1 does not. You just need to make it consistent.

Reply:
Put nullcheck in original places, and add an enforce here.